lib/membership/member.js

```diff
@@ -173,7 +173,7 @@ Member.prototype._applyUpdatePenalty = function _applyUpdatePenalty() {
 Member.prototype._isLocalOverride = function _isLocalOverride(update) {
```

Comment:
I was having a hard time understanding this function, would you mind adding this comment? https://github.com/uber/ringpop-node/pull/261